core/src/apps/common/signverify.py

```diff
@@ -41,21 +41,26 @@ def split_message(message: bytes) -> List[str]:
 
 
 async def require_confirm_sign_message(
-    ctx: wire.Context, header: str, message: bytes
+    ctx: wire.Context, coin: str, message: bytes
 ) -> None:
+    header = "Sign {} message".format(coin)
     message = split_message(message)
     text = Text(header, new_lines=False)
     text.normal(*message)
     await require_confirm(ctx, text)
 
 
 async def require_confirm_verify_message(
-    ctx: wire.Context, address: str, header: str, message: bytes
+    ctx: wire.Context, address: str, coin: str, message: bytes
 ) -> None:
     text = Text("Confirm address", new_lines=False)
+    text.bold("Coin: {}".format(coin))
+    text.br()
+    text.br_half()
```

Comment:
I am thinking "Confirm address" is very confusing. Maybe we could also have "Verify {} message" as the header and then text.bold("Confirm address:")?

Reply:
So it will look like this:
```
Verify ETH message

Confirm address:
ABCDEF
```

Reply:
Yeah that looks better - fixed.